src/Microsoft.TemplateEngine.Utils/EngineEnvironmentSettings.cs

```diff
@@ -13,10 +13,12 @@ namespace Microsoft.TemplateEngine.Utils
     public static class EngineEnvironmentSettings
     {
         private static readonly string DefaultLocale = "en_US";
+        private static readonly string DefaultAssemblyName = "None";
+        private static readonly Version DefaultAssemblyVersion = new Version("0.0.0.1");
 
         static EngineEnvironmentSettings()
         {
-            Host = new DefaultTemplateEngineHost(string.Empty, DefaultLocale);
+            Host = new DefaultTemplateEngineHost(string.Empty, DefaultLocale, DefaultAssemblyName, DefaultAssemblyVersion);
```

Comment:
Actually, we already do that in Program.cs, but we shouldn't have the default I don't think. This must always come from the host.